cpu/sam0_common/periph/spi.c

```diff
@@ -395,6 +396,7 @@ void spi_release(spi_t bus)
     /* Demux clk_pin back to GPIO_OUT function. Otherwise it will get HIGH-Z
      * and lead to unexpected current draw by SPI salves. */
     gpio_disable_mux(spi_config[bus].clk_pin);
+    gpio_disable_mux(spi_config[bus].mosi_pin);
```

Comment:
Since you removed the gpio_init code, are we sure `gpio_disable_mux` will go back in GPIO_OUT mode ?

Reply:
Ok, let's better leave this as is then.